### PR TITLE
🏗 Enable Error Reporting for SXG.

### PR DESCRIPTION
**Pull request by xiexr151e**
Small changes to enable error reporting for SXG build, based on the ESM build.

### CODE REVIEW

**Review by kevinkimball — APPROVED** [review]

**Review by erwinmombay — APPROVED** [review]

File: src/error.js [hunk 1]
```diff
@@ -544,6 +544,9 @@ export function getErrorReportData(
   if (IS_ESM) {
     runtime = 'esm';
     data['esm'] = '1';
+  } else if (IS_SXG) {
```
Comment (jridgewell) [inline src/error.js:547]: I think `IS_SXG` also implies `IS_ESM`, so this has to go first.

**Review by jridgewell — CHANGES_REQUESTED** [review]

File: src/error.js (hunk 1 above)
Comment (xiexr151e) [reply src/error.js]: Done.

Comment (erwinmombay) [reply src/error.js]: great catch. thanks @jridgewell

File: src/error.js [hunk 2]
```diff
@@ -541,12 +541,12 @@ export function getErrorReportData(
...
 
   let runtime = '1p';
-  if (IS_ESM) {
-    runtime = 'esm';
-    data['esm'] = '1';
-  } else if (IS_SXG) {
+  if (IS_SXG) {
     runtime = 'sxg';
     data['sxg'] = '1';
+  else if (IS_ESM) {
```
Comment (jridgewell) [inline src/error.js]:
```suggestion
  } else if (IS_ESM) {
```

**Review by jridgewell — APPROVED** [review]

**Pull request merged**